src/test/scala/cromwell/engine/db/slick/SlickDataAccessSpec.scala

```diff
@@ -262,9 +262,12 @@ class SlickDataAccessSpec extends FlatSpec with Matchers with ScalaFutures {
             new Call(callAlias, "call.name", task, Set.empty[FullyQualifiedName], Map.empty, None)
           }
 
+        val callKey = ExecutionDatabaseKey(call.fullyQualifiedName, None)
+        val shardKey = ExecutionDatabaseKey(call.fullyQualifiedName, Option(0))
```

Comment:
Maybe some constants for `None` and `Option(0)` in the context they're being used as indexes as these are repeated throughout the tests